Directory.Build.props

```diff
@@ -1,25 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Project>
   <PropertyGroup>
-    <NetVersion>net8.0</NetVersion>
-    <MauiPackageVersion>8.0.93</MauiPackageVersion>
+    <NetVersion>net9.0</NetVersion>
+    <MauiPackageVersion>9.0.0-rc.2.24503.2</MauiPackageVersion>
```

Comment:
Reminder to update this to the latest stable version of Microsoft.Maui.Controls before merging